v2/api/cache/v1beta20201201/redis_linked_server_types_gen.go

```diff
@@ -431,9 +431,7 @@ func (server *Redis_LinkedServer_Spec) PopulateFromARM(owner genruntime.Arbitrar
 	}
 
 	// Set property ‘Owner’:
-	server.Owner = &genruntime.KnownResourceReference{
-		Name: owner.Name,
-	}
+	server.Owner = &genruntime.KnownResourceReference{Name: owner.Name}
```

Comment:
This inline does not look right? it was fine earlier I guess.

Reply:
As discussed in Teams, will take this change (the new one-line way) as the consistency is nice and it's overall better when considering the use of all of our Composite Literals.